invisible_cities/reco/paolina_functions.py

```diff
@@ -350,30 +339,37 @@ def drop_voxel(voxels: Sequence[Voxel], the_vox: Voxel) -> int:
         for v in min_vs:
             v.E += the_vox.E/sum_en_v * v.E
 
-        return 1
+    def nan_energy(voxel):
+        voxel.E = np.nan
+        for hit in voxel.hits:
+            hit.Ep = np.nan
```

Comment:
Here the nan energy of the hit should be voxel.Etype

Reply:
Ok to all comments like this